### DISCUSSION TITLE
The media library is not working properly.

### DISCUSSION THREAD

**Discussion by dkoshy** [OrchardCMS/OrchardCore#7165] [Q&A]
Hi All,

After setting up the core  cms application , please navigate to media library in the Admin section . It will show "this folder is empty" message , It is not displaying default theme assets . When  user upload something , then only   it starts displaying the default assets.  It is minor issue. Please check.

**Comment by hishamco** [comment]
Is what you mean that the media library doesn't show the theme assets by default?

**Comment by jtkech** [comment]
@dkoshy I think it has been fixed in the dev branch through #6898
